client/js/views/LoginSignup.vue

```diff
@@ -122,9 +122,6 @@ export default {
       return 'Already have an account?';
     },
   },
-  mounted() {
-    this.$store.dispatch('requestLogout');
-  },
```

Comment:
I can't believe you've done this to me. I should have removed this as part of the login/logout refactor, hahaha.

Reply:
💋